product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx

```diff
@@ -20,14 +20,20 @@ psql -W "postgres://edb_admin@xxxxxxxxx.xxxxx.biganimal.io:5432/edb_admin?sslmod
 !!! Note
     Avoid storing data in the postgres system database.
 
-## Notes on the edb_admin role
+## Admin roles
 
-- You can grant the edb_admin role superuser privileges for a cluster. If you grant superuser privileges, you must take care to limit the number of connections used by superusers to avoid degraded service or compromising availability.
+### pg_ba_admin
+In order for us to effectively manage the cloud resources and ensure users are protected against security threats, BigAnimal provides a special administrative role, pg_ba_admin. The edb_admin user is a member of the pg_ba_admin role. The pg_ba_admin role has privileges similar to a Postgres superuser. Like the edb_admin user, the pg_ba_admin role shouldn't be be used for day-to-day application operations and access to the role must be controlled carefully. See [pg_ba_admin role](pg_ba_admin) for details.
```

Comment:
```suggestion
In order for us to effectively manage the cloud resources and ensure users are protected against security threats, BigAnimal provides a special administrative role, pg_ba_admin. The edb_admin user is a member of the pg_ba_admin role. The pg_ba_admin role has privileges similar to a Postgres superuser. Like the edb_admin user, the pg_ba_admin role shouldn't be be used for day-to-day application operations and access to the role must be controlled carefully. See [pg_ba_admin role](pg_ba_admin) for details.

The pg_ba_admin role is present on clusters deployed in BigAnimal's cloud account. Soon, it will be deployed to all BigAnimal clusters.
```